### PR TITLE
add some image classification model

### PR DESCRIPTION
**Pull request by LielinJiang**
add vgg, mobilenetv1, v2, resnet18, resnet34

### CODE REVIEW

File: models/mobilenet.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+import numpy as np
+import argparse
+import paddle
+import paddle.fluid as fluid
+from paddle.fluid.initializer import MSRA
+from paddle.fluid.param_attr import ParamAttr
+from paddle.fluid.layer_helper import LayerHelper
+from paddle.fluid.dygraph.nn import Conv2D, Pool2D, BatchNorm, Linear
+from paddle.fluid.dygraph.base import to_variable
+from paddle.fluid import framework
```
Comment (qingqing01) [inline models/mobilenet.py:28]: 1. Remove un-used importing, time, argparse , to_variable, framework....
2. Also need to clear for other files.

File: models/mobilenet.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+from model import Model
+from .download import get_weights_path
+
+__all__ = [
+    'MobileNetV2', 'mobilnetv2_x0_25', 'mobilnetv2_x0_5', 'mobilnetv2_x0_75',
+    'mobilnetv2_x1_0', 'mobilnetv2_x1_25', 'mobilnetv2_x1_5',
+    'mobilnetv2_x1_75', 'mobilnetv2_x2_0'
+]
+
+model_urls = {}
```
Comment (qingqing01) [inline models/mobilenet.py:39]: Maybe only needs on MobileNetv2 and put scale in init

Comment (qingqing01) [inline models/mobilenet.py:39]: Maybe only needs on MobileNetv2 and put scale in init ?

File: models/mobilenet.py [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+    def forward(self, inputs):
+        y = self._first_block(inputs, ifshortcut=False)
+        for inv_block in self._inv_blocks:
+            y = inv_block(y, ifshortcut=True)
+        return y
+
+
+class MobileNetV2(Model):
+    def __init__(self, class_dim=1000, scale=1.0):
```
Comment (qingqing01) [inline models/mobilenet.py:163]: Since all these models will be exported for users, need to add docs.

File: models/resnet.py [hunk 4]
```diff
@@ -11,7 +25,9 @@
 from model import Model
 from .download import get_weights_path
 
-__all__ = ['ResNet', 'resnet50', 'resnet101', 'resnet152']
+__all__ = [
+    'ResNet', 'resnet18', 'resnet34', 'resnet50', 'resnet101', 'resnet152'
```
Comment (qingqing01) [inline models/resnet.py:29]: resnet18, resnet34, ... -> ResNet18, ResNet34, ...

File: models/resnet.py [hunk 5]
```diff
@@ -92,16 +153,16 @@ def forward(self, inputs):
...
         x = fluid.layers.elementwise_add(x=short, y=conv2)
 
-        layer_helper = LayerHelper(self.full_name(), act='relu')
-        return layer_helper.append_activation(x)
-        # return fluid.layers.relu(x)
+        return fluid.layers.relu(x)
 
 
 class ResNet(Model):
     def __init__(self, Block, depth=50, num_classes=1000):
```
Comment (qingqing01) [inline models/resnet.py:160]: need docs for these models.

File: models/resnet.py [hunk 6]
```diff
@@ -175,6 +239,14 @@ def _resnet(arch, Block, depth, pretrained):
     return model
 
 
+def resnet18(pretrained=False):
```
Comment (qingqing01) [inline models/resnet.py:242]: Also need to set doc for ResNet18/34/...

File: models/mobilenet.py (hunk 1 above)
Comment (LielinJiang) [reply models/mobilenet.py]: done, thanks.

File: models/mobilenet.py (hunk 2 above)
Comment (LielinJiang) [reply models/mobilenet.py]: done, thanks.

Comment (LielinJiang) [reply models/mobilenet.py]: done. thanks.

File: models/mobilenet.py (hunk 3 above)
Comment (LielinJiang) [reply models/mobilenet.py]: done, thanks.

File: models/resnet.py (hunk 4 above)
Comment (LielinJiang) [reply models/resnet.py:29]: It is a function, so i think lower case will be good?

File: models/resnet.py (hunk 5 above)
Comment (LielinJiang) [reply models/resnet.py:169]: done, thanks

File: models/resnet.py (hunk 6 above)
Comment (LielinJiang) [reply models/resnet.py:251]: done, thanks.

File: image_classification/main.py [hunk 7]
```diff
@@ -1,4 +1,3 @@
-# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```
Comment (heavengate) [inline image_classification/main.py:1]: 这行是不是误删了？

Comment (LielinJiang) [reply image_classification/main.py:1]: 是的，感谢提醒，修复了。

**Pull request merged**